src/common/ads/ad.js

```diff
@@ -43,6 +46,9 @@ class Ad {
     this._inStream = options.inStream || false;
     this._vpaid = options.vpaid || false;
     this._streamId = options.streamId || '';
+    this._wrapperAdIds = options.getWrapperAdIds();
```

Comment:
options is [`PKAdOptions`](https://github.com/kaltura/playkit-js/blob/master/flow-typed/types/ad-options.js). and it's json with no these functions. seems like these lines were copied/pasted from ima by mistake.